app/views/govuk_publishing_components/components/_search.html.erb

```diff
@@ -1,8 +1,16 @@
 <%
-  size ||= ""
-  no_border ||= false
   shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
-  classes = %w(gem-c-search)
+
+  aria_controls ||= nil
+  button_text ||= "Search"
```

Comment:
Could this and line 7 be moved to translations? Just for best practice.

Reply:
Also done - thanks!